browser/src/control/Control.JSDialog.js

```diff
@@ -635,6 +639,7 @@ L.Control.JSDialog = L.Control.extend({
 		instance.startX = e.data.posx;
 		instance.startY = e.data.posy;
 		instance.updatePos = null;
+		instance.noOverlay = instance.id == 'autoFillPreviewTooltip';
```

Comment:
Please do not use hardcoded IDs in the Control.JSDialog (it's "framework" for all the dialogs).
Instead please put that `noOverlay` property into JSON of the popup you create :)

Reply:
I added `noOverlay: boolean;` into JSON and use this. I think it's ok :)
Thanks. Done.